src/modules/housingUnitType/dto/HouseUnitTypeFacilityCreate.dto.ts

```diff
@@ -1,14 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger'
 import { IsBoolean, IsDefined, IsOptional, IsString } from 'class-validator'
 
 export class HouseUnitTypeFacilityCreateDTO {
+    @ApiProperty({ example: 'true' })
```

Comment:
```suggestion
    @ApiProperty({ example: 'true', required: false })
```
campo não é obrigatório